src/index.js

```diff
@@ -59,3 +61,9 @@ formEl.addEventListener("submit", (event) => {
 });
 
 render();
+
+// Copy to clipboard ffmpeg build command 
+var clipboard = new ClipboardJS('#Copy');
+clipboard.on('success', function(e) {
+  document.getElementById("Copy").textContent="copied :)";  
```

Comment:
I'd like the element to show "copied" for a second, but then revert back. It should also revert back if you change any inputs.

Could you implement that?

Reply:
Implemented